src/main/java/com/github/skjolberg/packing/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.github.skjolberg.packing;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public interface Constants {
+
+	AtomicBoolean ALWAYS_FALSE = new AtomicBoolean(false);
```

Comment:
Could this constant perhaps go into the Packager instead - so no Constants interface?

Reply:
Yes, why not.